Source/ManagedNativeWifi/Win32/BaseMethod.cs

```diff
@@ -210,6 +210,38 @@ public static IEnumerable<WLAN_AVAILABLE_NETWORK> GetAvailableNetworkList(SafeCl
 			}
 		}
 
+		public static IEnumerable<BssEntryWrapper> GetNetworkBssEntryWrapperList(SafeClientHandle clientHandle, Guid interfaceId)
+		{
+			var wlanBssList = IntPtr.Zero;
+			try
+			{
+				var result = WlanGetNetworkBssList(
+					clientHandle,
+					interfaceId,
+					IntPtr.Zero,
+					DOT11_BSS_TYPE.dot11_BSS_type_any,
+					false,
+					IntPtr.Zero,
+					out wlanBssList);
+
+				// ERROR_INVALID_HANDLE: The client handle was not found in the handle table.
+				// ERROR_INVALID_PARAMETER: A parameter is incorrect. The interface is removed.
+				// ERROR_NOT_ENOUGH_MEMORY: Not enough memory is available to process this request.
+				// ERROR_NDIS_DOT11_POWER_STATE_INVALID: The interface is turned off.
+				// ERROR_NOT_FOUND: The inteface GUID could not be found.
+				// ERROR_NOT_SUPPORTED: The WLAN AutoConfig service is disabled.
+				// ERROR_SERVICE_NOT_ACTIVE: The WLAN AutoConfig service has not been started.

```

Comment:
Not completely sure how these errors occur. Are you saying that these are parsing errors that can occur?

Reply:
These are error codes that can be returned by the API call to WlanGetNetworkBssList. They are handled by the CheckResult method. I documented them primarily for consistency with similar code.